GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
```

Comment:
Subcommand - tasks performed under a specified rule. You typically find a subcommand when you have a command that can perform different tasks. For example, the `rules` command can perform a `list`, `create`, or `delete` task. These are the subcommands.